cmds/extract_strings.go

```diff
@@ -477,9 +482,29 @@ func (es *extractStrings) extractString(f *ast.File, fset *token.FileSet) error
 	shouldProcessBasicLit := true
 	ast.Inspect(f, func(n ast.Node) bool {
 		switch x := n.(type) {
+		case *ast.ExprStmt:
+			es.processEnforcedFunc(x.X, fset)
+		case *ast.AssignStmt:
+			for _, expr := range x.Rhs {
+				es.processEnforcedFunc(expr, fset)
+			}
+		case *ast.ReturnStmt:
+			for _, expr := range x.Results {
+				es.processEnforcedFunc(expr, fset)
+			}
+		case ast.Decl:
+			if gen, ok := x.(*ast.GenDecl); ok {
+				for _, spec := range gen.Specs {
+					if valSpec, ok := spec.(*ast.ValueSpec); ok {
+						for _, expr := range valSpec.Values {
+							es.processEnforcedFunc(expr, fset)
+						}
+					}
+				}
+			}
```

Comment:
Could this all be replaced by `case *ast.CallExpr`?

Reply:
🤯 it certainly could!